### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mykyta01**
- [DEMO LINK](https://mykyta01.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://mykyta01.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="/search"
+       method="GET"
+       data-qa="big"
+       class="search-bar">
```
Comment (Esceype) [inline src/index.html:17]: Checklist point number 5.
```suggestion
    <form
       action="/search"
       method="GET"
       data-qa="big"
       class="search-bar"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,41 @@
...
+         src="./images/Search.svg"
+         alt="Search icon"
+         class="search-bar__icon search-bar--small__icon"
+       >
+      <label>
+        <input type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        name="search"
+        class="search-bar__input search-bar--small__input"
+       >
```
Comment (Esceype) [inline src/index.html:47]: Add a new line for all attributes if their total number is more than three
```suggestion
        <input
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
          name="search"
          class="search-bar__input search-bar--small__input"
        >
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
+:root {
+  --searchBarColor: #3d4e61;
+  --borderColor: #e1e7ed;
+}
```
Comment (Esceype) [inline src/style.css:4]: Good

**Review by Esceype — CHANGES_REQUESTED** [review]
You did a great job, let's improve some points.

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,47 @@
...
+      <label class="find-form__label">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          name="search"
+          data-qa="keypress"
+          class="search-bar__input"
+        >
+      </label>
+    </form>
```
Comment (Andriy-Shymkiv) [inline src/index.html:34]:
```suggestion
    <form 
      action="/search" 
      method="GET" 
      data-qa="big" 
      class="search-bar"
    >
      <img 
        src="./images/Search.svg" 
        alt="Search icon" 
        class="search-bar__icon" 
      />

      <label class="find-form__label">
        <input 
          type="text" 
          placeholder='Try "Los Angeles"' 
          name="search" 
          data-qa="keypress" 
          class="search-bar__input"
        />
      </label>
    </form>
```

Comment (Andriy-Shymkiv) [reply src/index.html:34]: pay attention to your indents on this form and on below!

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
good job! left comments

File: src/index.html [hunk 5]
```diff
@@ -11,6 +11,48 @@
...
+       >
+      <label>
+        <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        name="search"
+        class="search-bar__input search-bar--small__input"
+       >
+      </label>
+    </form>
```
Comment (Andriy-Shymkiv) [inline src/index.html:56]: please fix indent on this form too!

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 4 above)
Comment (mykyta01) [reply src/index.html]: sry indents in css or in index.html code style?

**Pull request closed without merging**